soknad/src/main/java/no/nav/k9/søknad/ytelse/omsorgspenger/utvidetrett/v1/OmsorgspengerKroniskSyktBarn.java

```diff
@@ -32,6 +32,14 @@ public class OmsorgspengerKroniskSyktBarn implements OmsorgspengerUtvidetRett {
     @NotNull
     private Boolean kroniskEllerFunksjonshemming;
 
+    @JsonProperty(value = "høyereRisikoForFravær")
+    @Valid
+    private Boolean høyereRisikoForFravær;
+
+    @JsonProperty(value = "høyereRisikoForFraværBeskrivelse")
+    @Valid
+    private String høyereRisikoForFraværBeskrivelse;
```

Comment:
Fint om du legger på regex for fritekstfelt, slik andre fritekstfelt er satt opp i kontraktene :)